### DISCUSSION TITLE
Account Hacked with No Response from GitHub Support

### DISCUSSION THREAD

**Discussion by asmawi117** [community/community#109317] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

I first attempted to contact GitHub support regarding this matter precisely one week ago, and regrettably, I have yet to receive any acknowledgment or guidance on how to proceed.

**Comment by ghostinhershell** [comment]
Hey there! 👋

Thanks for posting in the GitHub Community, @asmawi117! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

With that being said, opening a support ticket is the best course of action for issues like this. Here are some resources on the process of [Creating a Support Ticket](https://docs.github.com/en/support/contacting-github-support/creating-a-support-ticket#about-support-tickets) and [How Ticket Priority is Determined](https://docs.github.com/en/enterprise-cloud@latest/support/learning-about-github-support/about-ticket-priority).

I've gone ahead and moved this to the correct category for you. Good luck!

**Reply by asmawi117** [reply]
It's been a week since I opened a support ticket and there has been no response from github support

**Comment by SamBeasley1026** [comment]
I'm currently running code for the internal revenue system! Anybody with good lawyers want to help me? I'll promise big pull!

**Comment by asmawi117** [comment]
35 days without answer from github support 😥

**Comment by asmawi117** [comment, marked as answer]
46 days and still being ghosted by github support

**Comment by asmawi117** [comment]
50 days and still being ghosted by github support
